recipes/proj-data/build.sh

```diff
@@ -0,0 +1,4 @@
+export DATADIR="${PREFIX}/share/proj"
```

Comment:
This should be updated related to this discussion: https://github.com/conda-forge/proj.4-feedstock/pull/71#issuecomment-596211051

Reply:
I started working on this but this clearly has to be a directory within the environment so it's not clear to me that `$XDG_DATA_HOME` can be of any use in this case.

Reply:
> I started working on this but this clearly has to be a directory within the environment so it's not clear to me that `$XDG_DATA_HOME` can be of any use in this case.

Not sure what we can do then. Makes sense that `conda` won't allow us to write this at build time.

Reply:
Right, and even if we could, it wouldn't end up in the package.